webpack/ForemanWebhooks/Routes/Webhooks/Components/WebhookForm/Components/FieldConstructor.js

```diff
@@ -0,0 +1,240 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { ExclamationCircleIcon, HelpIcon } from '@patternfly/react-icons';
+import {
+  TextInput,
+  Checkbox,
+  Button,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  Icon,
+  TextArea,
+  FormGroup,
+  Popover,
+} from '@patternfly/react-core';
+import AutocompleteInput from './AutocompleteInput';
+
+const FormField = ({
+  name,
+  type,
+  label,
+  required,
+  options,
+  isLoading,
+  validated,
+  value,
+  disabled,
+  setValue,
+  ...props
+}) => {
+  const [fieldValidated, setFieldValidated] = useState(validated);
+  const [firstLoad, setFirstLoad] = useState(true);
+
+  const requiredValidate = () => {
```

Comment:
The required is being validated, so when the field `Target URL` has a value it shown as valid, but if its not http/https (not sure what else the server checks) the form submit returns an error since `Target URL` is invalid. I think this can be misleading